src/components/cards/Item.tsx

```diff
@@ -72,17 +73,37 @@ function Item({ data, category }: any) {
                 } else {
                     newItems[catIndex].items.push(currentItem)
                     // send post to api to add new item
-                    client.post(`lists/${appConfig.activeListId}/items`, {
-                        item_id: id,
-                        list_id: appConfig.activeListId,
-                    })
+                    // Using async causes the setShopList to break
+                    client
+                        .post(`lists/${appConfig.activeListId}/items`, {
+                            item_id: id,
+                            list_id: appConfig.activeListId,
+                        })
+                        .then((response) => {
+                            toast.success('Item added to the list')
+                        })
+                        .catch((error) => {
+                            toast.error(
+                                'Something went wrong! List only updated locally'
+                            )
+                        })
```

Comment:
What did you have as an issue with async/await?

Reply:
By the way, the method addItemToShopList should be extracted somehow. I will also need it in the ShowItemSidebar. I don't know if using recoil to do some sideEffect to the shop list is the way to go? Otherwise, we could just the function or create a custom hook. Just throwing some random ideas :D